src/energyplus/ForwardTranslator/ForwardTranslateThermalZone.cpp

```diff
@@ -947,11 +948,13 @@ namespace energyplus {
                 }
               }
 
-              if (peopleSchedule) {
+              if (!allPeople.empty()) {
                 IdfObject zoneVentilation(IddObjectType::ZoneVentilation_DesignFlowRate);
                 zoneVentilation.setName(modelObject.name().get() + " Ventilation per Person");
                 zoneVentilation.setString(ZoneVentilation_DesignFlowRateFields::ZoneorZoneListName, modelObject.name().get());
-                zoneVentilation.setString(ZoneVentilation_DesignFlowRateFields::ScheduleName, peopleSchedule->name().get());
+                if (peopleSchedule) {
+                  zoneVentilation.setString(ZoneVentilation_DesignFlowRateFields::ScheduleName, peopleSchedule->name().get());
+                }
```

Comment:
Issue 3 on #4461: not translating it if you can't find the people schedule is wrong. You shouldn't try to write it if you didn't find it. You don't want to translate if you found zero people is all.